Resources/public/js/views/serverside/ez-contenttypeeditserversideview.js

```diff
@@ -109,12 +109,19 @@ YUI.add('ez-contenttypeeditserversideview', function (Y) {
         _reindexSelectionOptions: function (settingsNode) {
             var tplNode = this._getSelectionOptionPrototype(settingsNode),
                 inputIdTpl = tplNode.getData('ezselection-option-input-id'),
-                inputNameTpl = tplNode.getData('ezselection-option-input-name');
+                inputNameTpl = tplNode.getData('ezselection-option-input-name'),
+                isDefaultCheckboxNameTpl = tplNode.getData('ezselection-option-is-default-checkbox-name'),
```

Comment:
I don't like the name of variable `is...` suggests it is boolean.

Reply:
Yes, you are right. I will change that.